tools/aliaslookup/aliaslookup.py

```diff
@@ -20,9 +20,12 @@ def LookupByAliases(client: datastore.Client,
     verbose: a boolean whether to emit more verbose processing information.
 
   Returns:
-    a datastore.query.Query object
+    a datastore.query.Iterator
   """
   query = client.query(kind="Bug")
```

Comment:
nit: another comment for consistency. we generally use single quotes

Reply:
Huh. I'm surprised the linters didn't pick up on me being inconsistent. Fixed in https://github.com/google/osv.dev/pull/1198/commits/42551f1dbfc39578392cb7c4ca199a682ff0df4d and https://github.com/google/osv.dev/pull/1198/commits/59d5a48360e7a3b94505a95f2439f8f4d80d4706